crates/rpc/src/handlers/block.rs

```diff
@@ -16,6 +17,16 @@ use crate::types::{
     response::{BeaconResponse, BeaconVersionedResponse, RootResponse},
 };
 
+#[derive(Debug, Serialize, Deserialize, Default)]
+pub struct BlockRewards {
+    pub proposer_index: u64,
+    pub total: i64,
+    pub attestations: u64,
+    pub sync_aggregate: u64,
+    pub proposer_slashings: u64,
+    pub attester_slashings: u64,
```

Comment:
This isn't being serialized properly look at other code using `#[serde(with = "serde_utils::quoted_u64")]` for examples